parts/kuberneteskubelet.service

```diff
@@ -37,8 +37,6 @@ ExecStart=/usr/bin/docker run \
         --cluster-domain=cluster.local \
         --register-schedulable=${KUBELET_REGISTER_SCHEDULABLE} \
         --node-labels=${KUBELET_NODE_LABELS} \
-        --cloud-provider=azure \
```

Comment:
@colemickens seemed pretty set that this should be an option and not the default. That it would cause issues that a user would need to opt into.